lib/data/guides.json

```
@@ -213,6 +213,20 @@
     "lastEdited": "2020-07-02T06:29:55.000Z",
     "rank": 25
   },
+  {
+    "title": "Deploy Next.js and Userbase with Vercel",
+    "description": "Create a todo app with Next.js and Userbase and deploy it live with Vercel.",
```

```suggestion
    "description": "Create a Todo app with Next.js and Userbase and deploy it live with Vercel.",
```

I believe Todo is normally capitalized, see https://github.com/tastejs/todomvc